data/World/Ice Cavern.json

```diff
@@ -2,6 +2,9 @@
     {
         "region_name": "Ice Cavern Beginning",
         "dungeon": "Ice Cavern",
+        "locations":{
+            "Ice Cavern Frozen Blue Rupee": "Blue_Fire"
+        },
```

Comment:
Trick: You can roll or sidehop into this ice in the right way to collect the rupee. (Jumpslashes also work, which is the easier method as adult. Boomerang also works, both firing it directly at it or doing a loop around trick throw.)